client/src/pages/rantangan/components/RantanganContent.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Box } from '@mui/material';
+import GridContainer from 'shared/components/grid/GridContainer';
+import { Typography, DropdownMenu } from 'shared/components/index';
+import UPCOMING_RANTANGAN from 'shared/data/upcoming_rantangan';
+import bakmi from 'assets/rantangan-page/bakmi.svg';
+import PAST_RANTANGAN from './constants';
+import GridContent from '../../../shared/components/grid/GridContent';
+
+function RantanganContent({ upcoming }) {
+	const [selectedYear, setSelectedYear] = useState('2022/2023');
+	const eventData = upcoming ? UPCOMING_RANTANGAN : PAST_RANTANGAN[selectedYear];
+
+	return (
+		<Box className='my-20 relative'>
+			<Box className='absolute w-screen'>
+				{!upcoming && <img src={bakmi} alt='bakmi-icon' className='absolute w-[30%] right-10 z-[0] -top-80' />}
```

Comment:
<img width="283" alt="image" src="https://github.com/gisauweb/GISAUBC/assets/65900990/3a411db6-e1c2-49ef-b95d-0633ad57e9e5">

Can you ask Filbert, for mobile device, where should we put the murals?